src/components/rating/Rating.jsx

```diff
@@ -174,10 +174,12 @@ class Rating extends React.Component<RatingPropsType> {
     }
 
     return (
-      <div className={ratingClass} aria-label={label}>
+      <div className={ratingClass} aria-label={label} role="group">
```

Comment:
not sure it's the correct role for this component, wdyt?

Reply:
I don't think we need group here

Reply:
a11y tests are failing without this 'role' 
![image](https://user-images.githubusercontent.com/39903772/210748390-8d64e2c7-3406-4eb9-883a-42abadef4ee0.png)

Reply:
Oh I see, i am wondering why it didn't scream previously, maybe @katarzynatobis has some idea

Reply:
`group` doesn't seem to fit here. I would try a different approach - `<span className="sg-visually-hidden">{label}</span>` as a first child of that div